day_7/high_level.md

```diff
@@ -0,0 +1,37 @@
+## Caesar Cipher Write Up
+
+The Ceasar Cipher is a shift method at it's core.  Instead of substituting one letter for another, the method takes each letter in the string and shifts it a specified number of spaces to the left or right.  The first step is to define the overall shift method, and it's input string and number to shift.
+
+`def cipher(input, shift)`
+
+To keep track of how many spaces to shift, a way to track the current position of each letter in needed, then the number to shift over can be applied.  First, we break up the "input" string into individual characters.  I found the precise coding from google [primary reference link](https://medium.com/@alexander.virga/ruby-simple-string-encryption-shift-caesar-cipher-encoder-rot-9dedf06374d1):  
+
```

Comment:
Appreciate the transparency of showing your source of the solution you found. This is great.

Note: This exercise to get a step-by-step of how to solve the problem is to get you thinking algorithmically - like what is the recipe for making a caesar cipher? 

Explaining the code you found in plain language is great - excellent work. I'd challenge you in the future to try to think first about the steps as you'd explain them in plain language, and _then_ add in code snippets.